packages/autocomplete-plugin-recent-searches/src/createRecentSearchesPlugin.ts

```diff
@@ -50,19 +51,36 @@ export function createRecentSearchesPlugin({
             return store.getAll();
           },
           templates: {
-            item({ item }) {
-              return `
-                <div class="autocomplete-item">
-                  <div>
-                    <svg viewBox="0 0 22 22" width="16" height="16" fill="currentColor">
-                      <path d="M0 0h24v24H0z" fill="none"/>
-                      <path d="M13 3c-4.97 0-9 4.03-9 9H1l3.89 3.89.07.14L9 12H6c0-3.87 3.13-7 7-7s7 3.13 7 7-3.13 7-7 7c-1.93 0-3.68-.79-4.94-2.06l-1.42 1.42C8.27 19.99 10.51 21 13 21c4.97 0 9-4.03 9-9s-4.03-9-9-9zm-1 5v5l4.28 2.54.72-1.21-3.5-2.08V8H12z"/>
-                    </svg>
-
-                    <span>${item.query}</span>
-                  </div>
-                </div>
+            item({ item, root }) {
+              const icon = document.createElement('div');
+              icon.className = 'item-icon aa-RecentSearches--icon';
+              icon.innerHTML = `
+                <svg viewBox="0 0 22 22" width="20" height="20" fill="currentColor">
+                  <path d="M0 0h24v24H0z" fill="none"/>
+                  <path d="M13 3c-4.97 0-9 4.03-9 9H1l3.89 3.89.07.14L9 12H6c0-3.87 3.13-7 7-7s7 3.13 7 7-3.13 7-7 7c-1.93 0-3.68-.79-4.94-2.06l-1.42 1.42C8.27 19.99 10.51 21 13 21c4.97 0 9-4.03 9-9s-4.03-9-9-9zm-1 5v5l4.28 2.54.72-1.21-3.5-2.08V8H12z"/>
+                </svg>
               `;
+              const title = document.createElement('div');
+              title.className = 'aa-RecentSearches--title';
+              title.innerText = item.query;
+              const spacer = document.createElement('div');
+              spacer.className = 'aa-RecentSearches--spacer';
+              const deleteButton = document.createElement('button');
```

Comment:
The word "delete" feels a big strong, wdyt about "remove"?

YouTube uses this vocabulary:

![image](https://user-images.githubusercontent.com/6137112/94682930-fee49a80-0325-11eb-81a5-2fda15ce3b39.png)

Reply:
https://github.com/algolia/autocomplete.js/pull/326/commits/f4e08756a5a9f66de1240c7efdcb47053d5bfc65